onur/styles.css

```diff
@@ -0,0 +1,241 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    color:#18181a;
+}
+html {
+    font-family: "Open Sans", sans-serif;
+}
+/* navbar section */
+a {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navbar ul li a:hover {
+    text-decoration:underline;
+    color: black;
+}
+
+.text-center {
+    text-align: center;
+}
+.heading-small {
+    font-size: 1.2rem;
```

Comment:
Nice that you decided to use 'rem'! It would be good to explicitly set the font-size in the `html` selector (e.g. 16px). Setting font size explicitly when using rem helps ensure a consistent, accessible, and flexible design that can adapt to different user preferences and devices (some browsers might have different default font sizes).